core/store/directory.hpp

```diff
@@ -165,6 +166,7 @@ struct directory : private util::noncopyable {
   const directory_attributes& attributes() const noexcept {
     return const_cast<directory*>(this)->attributes();
   }
+
```

Comment:
why

Reply:
why what?

Reply:
You're added a lot of random empty lines in the PR.
Also removed some other empty lines.